float64 xPos
float64 yPos
bool show
---
string msg_feedback
